api/v3/Geometry.php

```diff
@@ -80,11 +98,24 @@ function civicrm_api3_geometry_delete($params) {
  * @throws API_Exception
  */
 function civicrm_api3_geometry_get($params) {
-  $results = _civicrm_api3_basic_get(_civicrm_api3_get_BAO(__FUNCTION__), $params);
+  if (!empty($params['format']) && !in_array($params['format'], ['json', 'kml', 'wkt'])) {
+    throw new API_Exception(E::ts('Output Format is not of an acceptable format it must be one of json, kml, or wkt'));
```

Comment:
"Output format must be one of json, kml or wkt".  Too wordy.

Reply:
Do you have an alternate suggestion? The acceptable values of format is 'json', 'kml', 'wkt' perhaps?

Reply:
My suggestion was as in my original comment "Output format must be one of json, kml or wkt"